FogCreek/Glitch-Community
# ginger-trout: even more styling fixes

## Links
* Remix link: https://ginger-trout.glitch.me/
* Manuscript link: 
https://glitch.manuscript.com/f/cases/3328544/Adjust-button-margins-underneath-embed
https://glitch.manuscript.com/f/cases/3328543/Adjust-bottom-margin-for-ProjectsContainer-to-add-spacing-between-container-report-abuse-button
https://glitch.manuscript.com/f/cases/3328541/Segmented-Button-Margins-are-off
https://glitch.manuscript.com/f/cases/3328542/Need-to-handle-overflow-of-markdown-in-project-descriptions

## Changes:
* Fixed overflow issue with markdown in project descriptions:
https://ginger-trout.glitch.me/search?q=saskia
https://glitch.com/search?q=saskia
![image](https://user-images.githubusercontent.com/519336/57795428-edd8c080-7713-11e9-9b98-ea5fa91daf7d.png)

* Fixed segmented button styling (note that there is an override in the add-project-to-collection-pop)
![image](https://user-images.githubusercontent.com/519336/57795468-12cd3380-7714-11e9-9b80-9e9044541ae9.png)

* Fixed margins for projects-list__projectsGrid, which was affecting the placement of the Report Abuse button (note there are now overrides when the projects-list is within a container, like more by or recent projects)
https://ginger-trout.glitch.me/@scientiffic-test
![image](https://user-images.githubusercontent.com/519336/57795613-6c356280-7714-11e9-9373-77151700af99.png)

* Adjusted collection-options-pop placement
![image](https://user-images.githubusercontent.com/519336/57795694-9a1aa700-7714-11e9-92b2-aea1a52446f5.png)

* Edited small button padding in response to https://glitch.manuscript.com/f/cases/3328544/Adjust-button-margins-underneath-embed

* Fixed grid size for collection pages' projectsGridCompact (currently showing a max of 2 columns on prod instead of 3)
![image](https://user-images.githubusercontent.com/519336/57795946-38a70800-7715-11e9-860e-7e8b70f95cde.png)


## How To Test:
* Confirm the changes listed above and browse to see if the changes might have affected anything else unintentionally?
